### PR TITLE
Create pwd chapter

### PR DESCRIPTION
**Pull request by jessicard**

### CODE REVIEW

**Comment by jessicard** [comment]
Maybe I should put an example of this command?

**Comment by jessicard** [comment]
@zee @kmanion Ready for review!

File: en-US/src/pwd.md [hunk 1]
```diff
@@ -0,0 +1,12 @@
+# pwd
+
+```
+pwd
+```
+
+This will tell you which directory you're currently in. In fact, `pwd` is short for "print working directory", which is asking the computer to tell you where you are currently. After pressing enter, your computer will answer your question.
+
```
Comment (zspencer) [inline en-US/src/pwd.md]: Where does "Answer your question" come from? Are we using "Question/Answer" as the mental model for running commands?

Comment (zspencer) [inline en-US/src/pwd.md]: Also, it may be better to explore the mental model before we start going into each command.

**Pull request closed without merging**